### ISSUE TITLE
New Devices : Notifications

### ISSUE THREAD

**Issue by suriyadeepan** [FSFTN/SC15RailsApp#11]
- Newly registered devices should receive previously sent notifications
- Suggestions
  - We can add a TTL (time to live) for each notification after which it becomes obsolete and doesn't have to be sent to new devices
  - Make the schedule dynamic, so when a new device sends a join request to server, the server can send back updated schedule to device as a special notification

**Comment by mohsalsaleem** [comment]
1. Working.
2.  
   -Aravinth suggested, it will be better if we could send all past notifications.
   My suggestion would be we can have TTL for notification in the app. So that old notifications are automatically deleted.

**Comment by suriyadeepan** [comment]
i agree... every message should have an expiration date... The Notification Table should contain a column for expiration date... Or it could be handled on the Client side.. Either way this feature is necessary...
